src/QirRuntime/lib/QIR/bridge-qis.ll

```diff
@@ -336,3 +347,8 @@ define double @__quantum__qis__arctan2__body(double %y, double %x) {  ; Q#: func
   %result = call double @quantum__qis__arctan2__body(double %y, double %x)
   ret double %result
 }
+
+define i64 @__quantum__qis__drawrandomint__body(i64 %min, i64 %max) {  ; operation DrawRandomInt (min : Int, max : Int) : Int      https://docs.microsoft.com/en-us/qsharp/api/qsharp/microsoft.quantum.random.drawrandomint

```

Comment:
Nit: maybe, put the comment above and split across two lines? (we are trying to keep the files in this project under 120 char wide)

Reply:
Fixing.

(Explanation: I try to keep the code dense such that more code fits in one screen page. In that regard I assume comments are not code, and it is ok if the comments go out of the screen page. The readers mostly read comments much more rare than the code, at least if the code is simple).